keps/sig-windows/5100-windows-dsr-and-overlay-support/README.md

```diff
@@ -967,6 +967,8 @@ We have not observed any additional failure modes with DSR or overlay networking
 
 ###### What steps should be taken if SLOs are not being met to determine the problem?
 
+N/A
```

Comment:
I'd suggest linking here to Troubleshooting section, rather than saying N/A, which contains pretty detailed information how to debug problems.